requirements-git.txt

```diff
@@ -2,3 +2,4 @@ git+https://github.com/coneoproject/COFFEE#egg=COFFEE
 git+https://github.com/firedrakeproject/ufl.git#egg=ufl
 git+https://github.com/firedrakeproject/fiat.git#egg=fiat
 git+https://github.com/FInAT/FInAT.git#egg=finat
+--recursive git+https://github.com/firedrakeproject/loopy.git#egg=loopy
```

Comment:
So this recursive here is just wrong... That option should be used when you want to pull another requirements file.